doi/src/main/java/ca/nrc/cadc/doi/DoiInitAction.java

```diff
@@ -251,12 +264,76 @@ private static MultiValuedProperties getConfig(boolean verify) {
                 sb.append("OK");
             }
         }
-        log.info(sb.toString());
 
         if (!ok) {
-            throw new IllegalStateException("incomplete config: " + sb.toString());
+            throw new IllegalStateException("incomplete config: " + sb);
         }
         return props;
     }
 
+    // check that the DOI parent node path, configured with the V0SPACE_PARENT_PATH_KEY property,
+    // exists and has the expected properties.
+    private static void checkParentFolders() {
+        MultiValuedProperties config = getConfig();
+        URI vospaceResourceID = URI.create(config.getFirstPropertyValue(DoiInitAction.VAULT_RESOURCE_ID_KEY));
+        String parentPath = config.getFirstPropertyValue(PARENT_PATH_KEY);
+
+        Subject adminSubject = SSLUtil.createSubject(new File("/config/doiadmin.pem"));
```

Comment:
Maybe the `README` should mention the exact location where the certs are expected?

Reply:
That would be helpful, fixed.